.travis.yml

```diff
@@ -11,6 +11,12 @@ after_success: "[[ $TRAVIS_PULL_REQUEST == \"false\" ]] && mvn deploy"
 
 sudo: false
 
+# https://github.com/travis-ci/travis-ci/issues/3259
+addons:
+  apt:
+    packages:
+      - oracle-java8-installer
```

Comment:
Is this necessary? I wonder if Travis has to install it on every build over and over again, making the build longer.